packages/perseus/src/widgets/number-line.tsx

```diff
@@ -472,25 +472,25 @@ class NumberLine extends React.Component<Props, State> {
         // it can't be interacted with.
         let fill;
         if (isOpen) {
-            fill = KhanColors._BACKGROUND;
+            fill = "transparent";
```

Comment:
This was changed based on a comment in the file defining `KhanColors` that said it should be made transparent instead of using `_BACKGROUND`